sconify.args

```diff
@@ -0,0 +1,13 @@
+--name=confidential-email-sender \
+--from=${IMG_FROM} \
+--to=${IMG_TO} \
+--binary-fs \
+--fs-dir=/app \
+--host-path=/etc/hosts \
+--host-path=/etc/resolv.conf \
+--binary=/usr/local/bin/python3.7 \
+--heap=1G \
+--dlopen=2 \
+--verbose \
+--no-color \
+--command="python3 /app/app.py"
```

Comment:
```suggestion
--command="python3 /app/run.py"
```

Reply:
Good catch! Let's rename the file so that it's similar to our other examples :)
See https://github.com/iExecBlockchainComputing/confidential-email-sender/pull/2/commits/41d2538c9eef0470b77d1be98cfab795ca609491.